components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -439,10 +440,13 @@ class DisplayToolbar internal constructor(
     /**
      * Updates the URL to be displayed.
      */
+    private var _url: URLStringUtils.DisplayUrl? = null
```

Comment:
And since we don't need hold state anymore in `DisplayUrl` this would become: 

```
 /**
     * Updates the URL to be displayed.
     */
    internal var url: CharSequence = ""
        set(value) {
            field = value
            views.origin.url = URLStringUtils.toDisplayUrl(value)
            updateIndicatorVisibility()
        }
```